### ISSUE TITLE
Uncaught TypeError: Cannot read property 'indexMetadata' of undefined

### ISSUE THREAD

**Issue by patvachon77** [joeferraro/MavensMate-Atom#150]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.180.0
**System**: Microsoft Windows 8.1
**Thrown From**: [MavensMate-Atom](https://github.com/joeferraro/MavensMate-Atom) package, v0.0.18

### Stack Trace

Uncaught TypeError: Cannot read property 'indexMetadata' of undefined

```
At C:\Users\patva_000\.atom\packages\MavensMate-Atom\node_modules\mavensmate\lib\mavensmate\commands\project\index-metadata.js:18

TypeError: Cannot read property 'indexMetadata' of undefined
  at Command.execute (C:\Users\patva_000\.atom\packages\MavensMate-Atom\node_modules\mavensmate\lib\mavensmate\commands\project\index-metadata.js:18:10)
  at [object Object].Client.executeCommand (C:\Users\patva_000\.atom\packages\MavensMate-Atom\node_modules\mavensmate\lib\mavensmate\client.js:225:26)
  at MavensMateCoreAdapter.executeCommand (C:\Users\patva_000\.atom\packages\MavensMate-Atom\lib\mavensmate-core-adapter.coffee:152:13)
  at atom-workspace.<anonymous> (C:\Users\patva_000\.atom\packages\MavensMate-Atom\lib\mavensmate.coffee:240:40)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\patva_000\AppData\Local\atom\app-0.180.0\resources\app\src\command-registry.js:243:29)
  at CommandRegistry.handleCommandEvent (C:\Users\patva_000\AppData\Local\atom\app-0.180.0\resources\app\src\command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (C:\Users\patva_000\AppData\Local\atom\app-0.180.0\resources\app\src\command-registry.js:156:19)
  at EventEmitter.<anonymous> (C:\Users\patva_000\AppData\Local\atom\app-0.180.0\resources\app\src\window-event-handler.js:69:30)
  at EventEmitter.emit (events.js:116:17)

```

### Commands

```
     -0:15.3 mavensmate:edit-project (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:01.7 mavensmate:index-metadata (ul.list-inline.tab-bar.inset-panel)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  },
  "MavensMate-Atom": {
    "mm_workspace": [
      "C:\\Users\\patva_000\\Google Drive\\workspace"
    ]
  }
}
```

### Installed Packages

```coffee
# User
MavensMate-Atom, v0.0.18

# Dev
No dev packages
```

**Comment by joeferraro** [comment]
can you please include steps to reproduce? @patvachon77

**Comment by joeferraro** [comment]
0.0.19 attempts to resolve this issue

**Comment by joeferraro** [comment]
this should be resolved in v1.0.0. please feel free to re-open if you're still experiencing this issue. thanks!
